src/reply.rs

```diff
@@ -44,10 +46,13 @@ use serde_json;
 
 use reject::Reject;
 // This re-export just looks weird in docs...
-pub(crate) use self::sealed::{ReplyHttpError, ReplySealed, Reply_, Response};
+pub(crate) use self::sealed::Reply_;
 #[doc(hidden)]
 pub use filters::reply as with;
 
+/// Response type into which types implementing the `Reply` trait are convertable.
+pub type Response = ::http::Response<Body>;
```

Comment:
Oh yea, I had forgotten that part of the reason the trait was sealed was to not expose the exact body type... oh well, not a big deal for 0.1.

Reply:
Ah, indeed. So, you have a plan for 0.2 to change this? E.g. by using an associated type?